client-hc/src/test/java/com/graphhopper/api/HeadingExamples.java

```diff
@@ -0,0 +1,81 @@
+package com.graphhopper.api;
+
+import com.graphhopper.GHRequest;
+import com.graphhopper.GHResponse;
+import com.graphhopper.util.Parameters;
+import com.graphhopper.util.shapes.GHPoint;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.Request;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Arrays;
+
+@Disabled
+public class HeadingExamples {
+    static GraphHopperWeb hopper;
```

Comment:
I'd rather not add disabled tests, also because the asserted distances will depend on the exact map we are using. We could move these tests into `GraphHopperTest`. Maybe heading is already covered sufficiently there.

Reply:
I deleted the test in client-hc, it was very redundant. In retrospect, it was more for me to understand in which ways Graphhopper can be called.